model-optimizer/extensions/front/EmbeddingBagFuse.py

```diff
@@ -0,0 +1,91 @@
+"""
+ Copyright (C) 2020 Intel Corporation
+
+ Licensed under the Apache License, Version 2.0 (the "License");
+ you may not use this file except in compliance with the License.
+ You may obtain a copy of the License at
+
+      http://www.apache.org/licenses/LICENSE-2.0
+
+ Unless required by applicable law or agreed to in writing, software
+ distributed under the License is distributed on an "AS IS" BASIS,
+ WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ See the License for the specific language governing permissions and
+ limitations under the License.
+"""
+
+from extensions.ops.embedding_bag import EmbeddingBagOffsetsSum
+from mo.front.common.replacement import FrontReplacementSubgraph
+from mo.graph.graph import Graph, rename_nodes
+
+
+class EmbeddingBagFuse(FrontReplacementSubgraph):
+    enabled = True
+
+    def run_after(self):
```

Comment:
If you want to use this transformation for other frameworks (for example, TF) you should run it after `TFSliceToSliceReplacer` transformation which creates Slice operation from TFSlice. However, it is unlikely that the pattern will be the same, because in TF the StridedSlice is usually used. So I recommend (not insist) to move this transformation to ONNX specific directory.